composer.json

```diff
@@ -5,7 +5,7 @@
   },
   "require": {
     "firebase/php-jwt": "^3.0",
-    "shopware/connect-sdk": "dev-master#v2.0.6"
+    "shopware/connect-sdk": "dev-CON-4756"
```

Comment:
just for testing and QA has to be changed to official release if SDK 2.0.7 is released